ipaserver/install/ca.py

```diff
@@ -54,8 +50,8 @@
 
 
 def subject_validator(valid_attrs, value):
-    if not isinstance(value, unicode):
-        v = unicode(value, 'utf-8')
+    if not isinstance(value, str):
+        v = str(value, 'utf-8')
     else:
         v = value
```

Comment:
I believe you can get rid of this if all together, for Python3 the current solution simply assigns `value` to `v`, is this causing any issues?

Reply:
Nevermind, low on coffee 😅